tests/lean/run/2389.lean

```diff
@@ -0,0 +1,35 @@
+/-!
+# Test verifying that nested predicates don't trigger the generation of `below` lemmas
+Since the case of nested predicates is not currently handled by `mkBelow`,
+trying to generate `Bar.below` triggers an error upon defining the inductive type.
+-/
+
+inductive Forall (P : α → Prop) : List α → Prop
+  | nil : Forall P []
+  | cons : {x : α} → P x → Forall P l → Forall P (x::l)
+
+inductive Tree : Type :=
+  | leaf : Nat → Tree
+  | node : List Tree → Tree
+
+set_option trace.Meta.IndPredBelow true
+
```

Comment:
```suggestion
set_option trace.Meta.IndPredBelow true in
```

Reply:
(This makes the expected output easier to understand, as it makes clear where the trace message should be coming from.)